CHANGELOG.md

```diff
@@ -4,6 +4,27 @@
 
 💥 Breaking changes:
 
+- Set `debounceFormSubmitTimer` at instance level instead of global
+
+  For buttons with `prevent-double-click` enabled we set the `debounceFormSubmitTimer` to each instance, so a specific action is stopped from being called multiple times without interfering with other submit buttons on the page.
+
+  To migrate you need to update the way you initialise the Button components from:
```

Comment:
This also means that if any buttons are added to the page _after_ the module has been initialised they'll no longer get this behaviour without being initialised separately.

Are we happy with that change? If so, can we be explicit about it in the changelog?

Reply:
That is right and it's missing from the changelog. Will update to reflect this. It is however aligned with how the rest of the components enhanced with JavaScript are being initialised.